tracim/tracim/templates/file/getone.mak

```diff
@@ -101,121 +101,36 @@
-                                if(page >= nb_page-1){
-                                    document.getElementById('next').disabled = true;
-                                }
-                                if(page <= 0){
-                                    document.getElementById('prev').disabled = true;
-                                }
-                            }
-                        </script>
-                    </td>
-                </tr>
-                <tr>
-                    <td class="tracim-title">${_('File')}</td>
-                    <td>
-                        <a href="${download_url}" tittle="${_('Download the file (last revision)')}">
-                            ${result.file.file.name}
-                            <span class="pull-right">
-                                ${ICON.FA_FW('fa fa-download')}
-                            </span>
-
-                        </a>
-                    </td>
-                </tr>
-                <tr>
-                    <td class="tracim-title">${_('Size')}</td>
-                    <td>${h.user_friendly_file_size(result.file.file.size)}</td>
-                </tr>
-                <tr>
-                    <td class="tracim-title">${_('Modified')}</td>
-                    <% created_localized = h.get_with_timezone(result.file.created) %>
-                    <td>${h.format_short(created_localized)|n} ${_('by {}').format(result.file.owner.name)}</td>
-                </tr>
-            </table>
+            <div id='previewGenerator'></div>
+            <script src="${tg.url('/assets/js/previewGenerator.vendor.bundle.js')}" type="text/javascript"></script>
+            <script src="${tg.url('/assets/js/previewGenerator.app.entry.js')}" type="text/javascript"></script>
+            <script type="text/javascript">
+              <% created_localized = h.get_with_timezone(result.file.created) %>
+              (function initialisePreviewGenerator() {
+                previewGenerator(
+                  document.getElementById('previewGenerator'),
+                  [
+                    % for one_url in url:
+                      '${one_url}',
+                    % endfor
+                  ],
+                  '${nb_page}',
+                  '/previews/__FILE_ID__/pages/__CURRENT_PAGE__/__DOWNLOAD_TYPE__?revision_id=__REVISION_ID__',
+                  {
+                    id: '${result.file.id}',
+                    name: '${result.file.file.name}',
+                    selectedRevision: '${result.file.selected_revision}',
+                    weight: '${h.user_friendly_file_size(result.file.file.size)}',
+                    height: '300',
+                    modifiedAt: '${h.format_short(created_localized)|n}',
+                    owner: '${result.file.owner.name}',
+                    sourceLink: '${download_url}',
+                    pdfAvailable: false // Côme - 2017/07/24 - backend should return this information
```

Comment:
'FIXME' added in https://github.com/tracim/tracim/pull/281/commits/eb6772c390f136c809a463d5c33f4265d5fbc379